src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +24,25 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        if (!Files.exists(Paths.get(fileName))) {
+            throw new FileNotFoundException();
+        }
+        String s = "";
+        s = Files.readString(Paths.get(fileName)).replaceAll("[^a-zA-Z]", "").toLowerCase();
```

Comment:
use Characters methods instead of replaceAll() https://mate-academy.github.io/jv-program-fulltime/01_core/checklist/hw_10.html